internal/cmd/login_test.go

```diff
@@ -61,8 +62,33 @@ func TestLoginCmdSignup(t *testing.T) {
 		exp.ExpectString(t, "Confirm")
 		exp.Send(t, "password\n")
 		exp.ExpectString(t, "Logged in as")
+	})
+
+	runStep(t, "login updated infra config", func(t *testing.T) {
```

Comment:
These two steps were moved to another test case in #2280, so I'm restoring them here.